src/Rule/Required.php

```diff
@@ -64,7 +65,7 @@ public function getNotPassedMessage(): string
     }
 
     /**
-     * @psalm-return EmptyCallback
+     * @return EmptyCallback
```

Comment:
```suggestion
     * @psalm-return EmptyCallback
```

`EmptyCallback` is psalm type only.

Reply:
Was unifying types (removing "psalm" prefixes) where it's possible, but accidentally changed the tag in this place too.

Reply:
Fixed.

Reply:
Not fixed, now `@return`.

Reply:
Weird, finally it's now fixed.